### ISSUE TITLE
Possible Contradiction between not enabling CORS and NuJS

### ISSUE THREAD

**Issue by afrancht** [nucypher/nucypher#1475]
TLDR: 
* I have found a contradiction between the recent decision of developing "NuCypherJS" (take it with a pinch of salt) and some of the conversations I've had with @jMyles and @vepkenez and the issues https://github.com/nucypher/nucypher/issues/383 and https://github.com/nucypher/nucypher/pull/893.

Explanation: 
Evidenced by @cygnusv in https://github.com/nucypher/nucypher/issues/383
> We went for option 3 (creating a REST interface for characters) at that moment. But we may complement that with other approaches too.
>

In answering the question **how we will initially implement javascript support?** 

There seems to be a contradiction with #893 in which the decision was: 
> If the character endpoints are open and can be accessed from any domain, don't you expect that other domains will attack these endpoints (presumably via malicious javascript)?
>

Additionally, @vepkenez and I have been talking about Enrico, Bob and AliceJS, which raises the questions:
- Won't these character implementations be making API calls without being executed from within a python process?
- Won't this need CORS enabled?

Current in place solution:
* I know you are/have developed a NativeMessaging Firefox Extension to make the NuCypher network more usable, but this may still not be in line with how Alice/Bob/EnricoJS will work. But that is a discussion that can be had in #1474 

I'm sure you have already thought about this and figured out how you're going to do it but
for this issue, I was just wondering what the decision/outcome is/will be?

**Comment by KPrasch** [comment]
Thank you for articulating these ideas -- it was ahead of it's time! With the advent of nucypher/nucypher-ts this issue can be closed.  Please direct future issues to the new repo. 🙂
